actions/lib/actions.py

```diff
@@ -55,8 +55,11 @@ def invoke(self, func, *args, **kwargs):
                 # NOTE: the default page number = 1
                 params['page'] = page_number
 
-            # invoke the Cloudflare APIo
-            raw_results = func(*args, params=copy.deepcopy(params))
+            # invoke the Cloudflare API
+            if func.__name__ == 'post':
```

Comment:
Would like to see 2 tests to show the different conditions are being executed as expected. https://github.com/StackStorm-Exchange/stackstorm-cloudflare/blob/master/tests/test_action_base.py